src/main/java/apollo/maven/MavenDeployer.java

```diff
@@ -31,6 +31,7 @@ public void deployArtifact(Path pathToPom) {
         GAV gav = gavFactory.createGAV(pathToPom);
         logger.debug("Executing command - " + commandToExecute);
         invocationRequest.setGoals(Collections.singletonList(commandToExecute));
+        invocationRequest.setMavenOpts("-client -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xverify:none");
```

Comment:
Can you explain or  maybe leave a comment why this options are necessary

Reply:
Added comment